app/RealopsFlight.php

```diff
@@ -42,4 +42,8 @@ private function formatTime($time) {
         $time_split = explode(':', $time);
         return $time_split[0] . ':' . $time_split[1];
     }
+
+    public function getImageDirectory() {
+        return '/photos/airline_logos/' . substr($this->flight_number, 0, 3) . ".png";
```

Comment:
@Evanston09 you need to add some protection in here to deal with cases where `'/photos/airline_logos/' . substr($this->flight_number, 0, 3) . ".png"` is not a valid file. I recommend that you add either a default logo of an airplane silhouette or use a small transparent logo as the default so it doesn't break the blade.

Reply:
Well done!